src/test/java/com/shippingcostestimator/enterprise/EnterpriseApplicationTests.java

```diff
@@ -99,10 +102,13 @@ private void whenShipmentWithId1Exists(){
         shipment.setCountryTo("USA");
         shipment.setZipTo("54321");
 
-        shipment.setLength(20.2);
-        shipment.setWidth(10.5);
-        shipment.setHeight(9.9);
-        shipment.setWeight(100);
+        packageInfo.setLength(20.2);
+        packageInfo.setWidth(10.5);
+        packageInfo.setHeight(9.9);
+        packageInfo.setWeight(100);
+
+        shipment.setPackageInfo(packageInfo);
+
```

Comment:
Since I made changes to the class. I updated your test to reflect these changes. It also shows how this new implementation would work.

Reply:
Excellent!  Complete follow through.